weblate/vcs/git.py

```diff
@@ -621,14 +641,7 @@ def get_merge_message(self):
 class GithubRepository(GitMergeRequestBase):
 
     name = "GitHub"
-    _cmd = "hub"
     _version = None
-    req_version = "2.7"
-
-    @classmethod
-    def _get_version(cls):
-        """Return VCS program version."""
-        return cls._popen(["--version"], merge_err=False).split()[-1]
```

Comment:
This is what I meant by https://github.com/WeblateOrg/weblate/pull/4224/files#r462265471, sorry for not doing a pull request, I've messed it up somehow.

Reply:
No problem